portal/views.py

```diff
@@ -145,31 +145,36 @@ class HomeView(View):
     def get(self, request, app_id=0):
         published_apps, request = get_public_apps(request, app_id=app_id, get_all=False)
         published_models = PublishedModel.objects.all()
-        news_objects = NewsObject.objects.all().order_by("-created_on")
-        for news in news_objects:
-            news.body_html = markdown.markdown(news.body)
-        link_all_news = False
         if published_models.count() >= 3:
             published_models = published_models[:3]
         else:
             published_models = published_models
 
+        news_objects = NewsObject.objects.all().order_by("-created_on")
+        link_all_news = False
```

Comment:
I guess it's a link to all news?

Reply:
Yes, indeed, "link all news?". I changed a bit to so that it is consistent between news and events.